nautobot_ssot_infoblox/diffsync/adapters/infoblox.py

```diff
@@ -5,7 +5,8 @@
 from diffsync import DiffSync
 from diffsync.enum import DiffSyncFlags
 from nautobot.extras.plugins.exceptions import PluginImproperlyConfigured
-from nautobot_ssot_infoblox.diffsync.client import InfobloxApi
+from nautobot_ssot_infoblox.utils.client import InfobloxApi
```

Comment:
I am curious why the client was moved, is this following a new standard?

Reply:
To align with the new SSoT cookie and I believe Django organizational guidelines utility files should go in a utils folder. This cleans up the structure and clarifies the use of what's in the file IMO.